### ISSUE TITLE
[Frontend] AttributeError: 'NoneType' object has no attribute 'requirements' during CFG creation

### ISSUE THREAD

**Issue by mm4rks** [fkie-cad/dewolf#173]
### What happened?

possibly related to #150 

```python
Traceback (most recent call last):
  File "/home/user/dewolf/decompile.py", line 82, in <module>
    main(Decompiler)
  File "/home/user/dewolf/decompiler/util/commandline.py", line 81, in main
    undecorated_code = decompiler.decompile_all(options)
  File "/home/user/dewolf/decompile.py", line 71, in decompile_all
    task = self._frontend.create_task(function, task_options)
  File "/home/user/dewolf/decompiler/frontend/binaryninja/frontend.py", line 137, in create_task
    raise e
  File "/home/user/dewolf/decompiler/frontend/binaryninja/frontend.py", line 126, in create_task
    cfg = self._extract_cfg(function.function, options)
  File "/home/user/dewolf/decompiler/frontend/binaryninja/frontend.py", line 156, in _extract_cfg
    return parser.parse(function)
  File "/home/user/dewolf/decompiler/frontend/binaryninja/parser.py", line 34, in parse
    index_to_BasicBlock[basic_block.index] = BasicBlock(basic_block.index, instructions=list(self._lift_instructions(basic_block)))
  File "/home/user/dewolf/decompiler/structures/graphs/basicblock.py", line 38, in __init__
    self._update()
  File "/home/user/dewolf/decompiler/structures/graphs/basicblock.py", line 183, in _update
    for dependency in instruction.requirements:
  File "/home/user/dewolf/decompiler/structures/pseudo/instructions.py", line 142, in requirements
    return self._destination.requirements + self._value.requirements
  File "/home/user/dewolf/decompiler/structures/pseudo/operations.py", line 226, in requirements
    return self._collect_required_variables(self.operands)
  File "/home/user/dewolf/decompiler/structures/pseudo/operations.py", line 246, in _collect_required_variables
    return list(InsertionOrderedSet(chain(*[op.requirements for op in operands])))
  File "/home/user/dewolf/decompiler/structures/pseudo/operations.py", line 246, in <listcomp>
    return list(InsertionOrderedSet(chain(*[op.requirements for op in operands])))
```

### How to reproduce?

```bash
python decompile.py 7923f949e7422ac02c2dc5148950861f8102c83859f00b7d09857b95f16f7caf sub_40321f --pipeline.debug
``` 

### Affected Binary Ninja Version(s)

3.3.3996 (Build ID e34a955e)

**Comment by NeoQuix** [comment]
Will be fixed with #151.
Has the `MLIL_CONST_PTR` bug, where not the address but the value is used (0xC8).

**Comment by NeoQuix** [comment]
Fixed with #150
